pkg/webservice/routes.go

```diff
@@ -122,6 +122,18 @@ var webRoutes = routes{
 		"/ws/v1/partition/:partition/queue/:queue/application/:application",
 		getApplication,
 	},
+	route{
+		"Scheduler",
+		"GET",
+		"/ws/v1/partition/:partition/application/:application",
+		getApplication,
+	},
+	route{
+		"Scheduler",
+		"GET",
+		"/ws/v1/partition/:partition/applications/:state",
```

Comment:
this endpoint is under discussion. https://issues.apache.org/jira/browse/YUNIKORN-2235

Reply:
Updated jira with more details. Please check.